setup.py

```diff
@@ -3,6 +3,15 @@
 setuptools.setup(
     author="Allen Goodman",
     author_email="allen.goodman@icloud.com",
+    extras_require={
+        "test": [
+            "codecov",
+            "pytest",
+            "pytest-cov",
+            "pytest-pep8",
+            "pytest-xdist"
```

Comment:
I like xdist, but we could probably forego adding it as a dependency so it’s unused.